patterns/archive-writer.php

```diff
@@ -10,47 +10,19 @@
 
```

Comment:
We probably should add page to the template types

Reply:
Why would the archive template be suggested when you are creating a page template?